src/braket/device_schema/oqc/oqc_provider_properties_v1.py

```diff
@@ -73,4 +69,4 @@ class OqcProviderProperties(BraketSchemaBase):
         name="braket.device_schema.oqc.oqc_provider_properties", version="1"
     )
     braketSchemaHeader: BraketSchemaHeader = Field(default=_PROGRAM_HEADER, const=_PROGRAM_HEADER)
-    properties: Dict[str, Dict[str, QubitType]]
+    specs: Dict[str, Dict[str, QubitType]]
```

Comment:
I'm concerned this is backwards incompatible change. Is this currently breaking something, or just a wart? What testing has been done to ensure that we aren't breaking things in- or externally?